src/main/java/billgates/view/gui/ActionPanel.java

```diff
@@ -221,7 +224,11 @@ private void backFromSplit() {
     }
 
     private void addEntry() {
-        JTextField dateField = new JTextField(20);
+        // Create a date picker that begin with Monday.
+        DatePickerSettings dateSettings = new DatePickerSettings();
```

Comment:
We need a date time picker, instead of only a date picker.
I believe this library provides both.